build_projects/shared-build-targets-utils/Utils/HostVersion.cs

```diff
@@ -49,7 +49,7 @@ public string ToString()
         //
 
         // Full versions and package information.
-        public string LatestHostPrerelease => "rc3";
+        public string LatestHostPrerelease => ReleaseSuffix;
```

Comment:
We're going to have to clean up this version design. Tracking what is going where takes some effort